pages/ksphere/kommander/1.1.0-beta/operations/infrastructure-providers/configure-aws-infrastructure-provider-roles/index.md

```diff
@@ -199,17 +201,28 @@ EOF
 }
 ```
 
+### Create Infrastructure Provider
+
 In Kommander, select the Workspace associated with the credentials you are adding.
 
-Navigate to **Administration > Cloud Providers** and click the **Add Cloud Provider** button.
+Navigate to **Administration > Infrastructure Providers** and click the **Add Infrastructure Provider** button.
+
+![Adding an Infrastructure Provider](/ksphere/kommander/1.1.0-beta/img/empty-infrastructure-providers.png)
+<br />_Adding an Infrastructure Provider_
 
-![Add Cloud Provider](/ksphere/kommander/1.1.0-beta/img/add-cloud-provider.png)
+![Add Infrastructure Provider Form](/ksphere/kommander/1.1.0-beta/img/add-infrastructure-provider.png)
+<br />_Add Infrastructure Provider Form_
 
-- Select the Amazon Web Services (AWS) option from the Add Cloud Provider.
+- Select the Amazon Web Services (AWS) option from the Add Infrastructure Provider.
 - Ensure "Role" is selected as the Authentication Method.
-- Select a name for your cloud provider. Consider choosing a name that matches the AWS user.
+- Enter a name for your infrastructure provider. Consider choosing a name that matches the AWS user.
```

Comment:
```suggestion
- Enter a name for your infrastructure provider. Select a name that matches the AWS user.
```

Reply:
Selecting a name that matches the AWS user is a recommendation, not a requirement as far as I know.